mkosi/__init__.py

```diff
@@ -5175,7 +5182,8 @@ def build_stuff(args: CommandLineArguments) -> None:
 
     # Make sure tmpfiles' aging doesn't interfere with our workspace
     # while we are working on it.
-    with open_close(workspace.name, os.O_RDONLY|os.O_DIRECTORY|os.O_CLOEXEC) as dir_fd:
+    with open_close(workspace.name, os.O_RDONLY|os.O_DIRECTORY|os.O_CLOEXEC) as dir_fd,\
```

Comment:
I do agree, that the line is a bit long, but I'd put it on one line to omit the line continuation. It's a bit awkward to read otherwise.